run_robot_test.sh

```diff
@@ -168,6 +168,13 @@ case "$(uname -s)" in
 esac
 
 
-./venv/bin/robot  -d "${TEST_ARTIFACT_DIR}" -x xunit_test_result.xml -r test_report.html ${TEST_VARIABLES} --variablefile ${TEST_VARIABLES_FILE} ${EXTRA_ROBOT_ARGS} ${TEST_CASE_FILE}
+./venv/bin/robot  ${EXTRA_ROBOT_ARGS} -d ${TEST_ARTIFACT_DIR} -x xunit_test_result.xml -r test_report.html ${TEST_VARIABLES} --variablefile "${TEST_VARIABLES_FILE}" "${TEST_CASE_FILE}"
```

Comment:
On the variable expansion, shall we follow same format everywhere without using double quotes around the variable? LMKWYT

Reply:
I agree, the standard and best way to use variables is inside double quotes to avoid incorrect expansion. In our case though, problem is that it can only be used for non-empty variables
If empty string variables are put inside double quote, robot tries to interpret them as command line input and fails to run.

I should, however, add the double quotes for ${TEST_ARTIFACT_DIR}
Thank you!